react/Dockerfile

```diff
@@ -1,30 +1,19 @@
-FROM node:lts AS development
+## build stage ##
+FROM node:lts-alpine3.21 AS build
 
 WORKDIR /app
 
-COPY package*.json ./
-RUN npm ci
-
 COPY . .
 
-ENV CI=true
-ENV PORT=3000
-
-CMD [ "npm", "start" ]
-
-FROM development AS builder
+RUN npm install
```

Comment:
`npm ci` fit better for Docker build: https://docs.npmjs.com/cli/v11/commands/npm-ci